docs/overview.asciidoc

```diff
@@ -9,14 +9,13 @@ endif::[]
 
 [partintro]
 --
-Built on the Kubernetes Operator pattern, Elastic Cloud on Kubernetes (ECK) extends the basic Kubernetes orchestration capabilities to support the setup and management of Elasticsearch, Kibana and APM Server on Kubernetes.
+Built on the Kubernetes Operator pattern, Elastic Cloud on Kubernetes (ECK) extends the basic Kubernetes orchestration capabilities to support the setup and management of Elasticsearch, Kibana, APM Server, Enterprise Search, and Beats on Kubernetes.
 
-With Elastic Cloud on Kubernetes you can streamline all those critical operations, such as:
+With Elastic Cloud on Kubernetes you can streamline critical operations, such as:
 
 . Managing and monitoring multiple clusters
-. Scaling cluster capacity up and down
-. Changing cluster configuration
-. Scheduling backups
+. Scaling cluster capacity and volume
+. Configure backups using snapshots
```

Comment:
I think I would maybe remove that item since we don't really have first-class support for snapshots. @elastic/cloud-k8s @shubhaat any different opinion?

Reply:
Here we just picked a single feature of ES and we emphasise that we support it. I think we could either:
1. make a broader statement listing more "big" features (but I'm not sure what should go into that list, we would need to refresh it etc.), or
2. make a general statement that we support all Stack features (but I'm not sure if we can say that about every single feature/setting that ES supports), or
3. as @sebgl suggests, remove this